nixos/modules/services/security/authelia.nix

```diff
@@ -71,14 +70,6 @@ let
               '';
             };
 
-            oidcIssuerPrivateKeyFile = mkOption {
-              type = types.nullOr types.path;
-              default = null;
-              description = ''
-                Path to your private key file used to encrypt OIDC JWTs.
-              '';
-            };
```

Comment:
Probably better to leave in the defunct option, than to remove the dead assertion.

I think we can mark it as `internal`, so that it doesn't show up in the option reference any more.